liferay-lima/liferay-portal
# LPS-194396 Show error if AL/Site are not connected

https://liferay.atlassian.net/browse/LPS-194396

How to test it


1. Create an Asset Library
2. Create a document
3. Create a shortcut
4. Create a folder
5. try to copy to a site (liferay or _Global_)

✔️ error message is thrown
![image](https://github.com/liferay-lima/liferay-portal/assets/47524751/be3d833e-e6ed-4ca5-930e-9a74224cd56c)


1. Connect to a site
2. try to copy to _the connected site_

✔️ item copied


- LPS-194396 if group does not exist should fail
- LPS-194396 check on the copy of the file if the destination or the source is an asset library, and only one of them, if they are connected. If not throw an exception
- LPS-194396 check on the copy of the shortcut if the destination or the source is an asset library, and only one of them, if they are connected. If not throw an exception
- LPS-194396 check on the copy of the folder if the destination or the source is an asset library, and only one of them, if they are connected. If not throw an exception
- LPS-194396 buildLang
